.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,40 @@
-#### What does this PR do?
-#### Any background context you want to provide?
 #### What ticket does this PR close?
 Connected to [relevant GitHub issues, eg #76]
-#### Where should the reviewer start?
-#### How should this be manually tested?
-#### Screenshots (if appropriate)
-#### Has the Version and Changelog been updated?
-#### Questions:
-> Does this work have automated integration and unit tests?
 
-> Can we make a blog post, video, or animated GIF of this?
+## Checklist
+### Code
+- [ ] The code works.
+- [ ] The code performs its intended function and the logic is correct.
```

Comment:
Would this cover `The code works` as well? Or is there a distinction between the two that should be called out?

Reply:
But I also suggest that we would have a section that requests from the engineer to give a small description on the PR, explain what was the intended functionality and if there is any unchecked box left, justify it. I think that rarely there will be good reasons to leave some of them unchecked.